Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM golang:1.7.3
+RUN curl https://glide.sh/get | sh
+WORKDIR /mine/src/github.com/kubernetes-incubator/service-catalog
```

Comment:
is `mine` a widely used convention?

Unless there is strong desire to deviate, might we go with:
https://github.com/kubernetes-incubator/service-catalog/blob/master/hack/Makefile.mk#L86 ?